Makefile

```diff
@@ -273,14 +275,24 @@ e2e-dependencies:
 
 K8SCLIENT ?= oc
 GINKGO = $(LOCAL_BIN)/ginkgo
+IS_HOSTED ?= false
+CLUSTER_NAMESPACE_ON_HUB =
+TIMEOUT_SECONDS = 30
 .PHONY: e2e-test
 e2e-test:
 	@if [ -z "$(TEST_FILE)" ]; then\
-		$(GINKGO) -v --no-color $(TEST_ARGS) --fail-fast test/e2e -- -cluster_namespace=$(MANAGED_CLUSTER_NAME) -k8s_client=$(K8SCLIENT) ;\
+		$(GINKGO) -v --no-color $(TEST_ARGS) --fail-fast test/e2e -- -cluster_namespace=$(MANAGED_CLUSTER_NAME) -k8s_client=$(K8SCLIENT) -is_hosted=$(IS_HOSTED) -cluster_namespace_on_hub=$(CLUSTER_NAMESPACE_ON_HUB) -timeout_seconds=$(TIMEOUT_SECONDS);\
```

Comment:
In hosted mode, takes more time than normal more so I had to increase this number